.NET/Microsoft.Recognizers.Definitions/Microsoft.Recognizers.Definitions.csproj

```diff
@@ -1,241 +1,250 @@
-<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Was this just a line break change? If so, please revert it and only add the lines you need.
Thanks.